gravity_norm: 9.831 # Norm in m/s^2, sign and reference frame is sorted in imu.cpp
lever_arm: [0, 10, -17]

# Uncertanties on initial values (The priors on veocity and biases will be removed before final optimization)
initial_velocity_sigma: [2, 2, 0.05]
initial_attitude_sigma: 0.5
initial_imu_bias_sigma: [0.1, 0.1, 0.1, 0.01, 0.01, 0.01] # ax, ay, az, gx, gy, gz

# IMU noise parameters
imu_accel_noise_sigma: 2.0e-3         # Accelerometer noise (continous)
imu_gyro_noise_sigma: 2.0e-4          # Gyro noise (continous)
imu_accel_bias_rw_sigma: 2.0e-4       # No bias drift
imu_gyro_bias_rw_sigma: 5.0e-6        # No bias drift

imu_split_integration: true
imu_max_integration_interval: 1.5

# GNSS stuff
gnss_sigma: 2                 # TODO: Sigma of measurement error
gnss_sample_interval: 1       # TODO: Downsample GNSS (Cause its noisy)

gnss_crs_source: "EPSG:4326"
gnss_crs_target: "EPSG:6052"

# Virtual height
virtual_height_interval: 1
virtual_height_sigma: 0.1
virtual_height_levered: true

altitude_estimate_gm: false # Use gauss-markov factor to constrain altitude
altitude_gm_tau: 30
altitude_gm_sigma: 0.1 # Use a rough estimate of root mean square

optimize_interval: 20 # Sub-problem optimization interval
optimize_remove_priors: true